examples/drupal_loadtest.rs

```diff
@@ -99,25 +103,34 @@ fn drupal_loadtest_front_page(client: &mut GooseClient) {
         },
     }
 }
+fn drupal_loadtest_front_page_async<'r>(client: &'r mut GooseClient) -> Pin<Box<dyn Future<Output = ()> + Send + 'r>> {
```

Comment:
In an ideal world, we'd get rid of this wrapper function entirely before merging, but if we can't we probably want to rename it.

I suspect that the suffix `_async` comes from Darksonn/Alice's playground here https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=938b5daebbfe9b86125f8731aeb2302a . There are a bunch of things that are misleadingly named in that playground, so I renamed a few of them: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ddeda28da92f8e0425536af9bf3800a9

Reply:
Yes, these wrappers would be very confusing to anyone using Goose, I'm 100% committed to at least hiding them behind a macro before we merge this. And you are correct, I copied Alice's naming but agree it's backwards.

Reply:
The wrapper issue is now resolved thanks to `task!()`.